### DISCUSSION TITLE
Wrong heavy distortion during phase-field fracture test

### DISCUSSION THREAD

**Discussion by rh201** [idaholab/moose#23863] [Q&A Modules: Solid mechanics] [answered]
Dear MOOSEers,

When I'm doing phase-field fracture with a tensile tension example as shown left below (fixed bottom and applied tension at top),  the simulation result shows the mesh is heavily distorted, which is totally unreasonable.  The input files are as follows https://github.com/rh201/Phase-field-ductile-fracture/blob/main/elasticity.i. Any comments and suggestions are highly appreciated. Thanks.

![Picture1](https://user-images.githubusercontent.com/79764031/228036248-bd518ea9-185e-47e9-a7ba-e619517c6c3f.png)

**Comment by GiudGiud** [comment, marked as answer]
hello

Are you expecting to be able to simulate the rupture of sample or do you think it should not reach this situation?

Guillaume

**Reply by rh201** [reply]
Hi yes. The sample is expected to fracture in the middle, as reported in the literature.

**Reply by GiudGiud** [reply]
@jiangwen84 @dschwen 
You ll want to look at removing elements to simulate the fracture.

**Reply by rh201** [reply]
With the phase-field fracture method, the elements are not need to be removed. 
BTW, when I don't consider plasticity (simply, brittle fracture), it works ok.
